CONTRIBUTING.md

```diff
@@ -298,10 +298,10 @@ The rules:
 1. All code should be formatted with `gofmt -s`.
 2. All code should pass the default levels of
    [`golint`](https://github.com/golang/lint).
-3. All code should follow the guidelines covered in [Effective
+3. All codes should follow the guidelines covered in [Effective
    Go](http://golang.org/doc/effective_go.html) and [Go Code Review
    Comments](https://github.com/golang/go/wiki/CodeReviewComments).
-4. Comment the code. Tell us the why, the history and the context.
+4. Comment on the code. Tell us the why, the history and the context.
```

Comment:
```suggestion
4. Include code comments. Tell us the why, the history and the context.
```
I realize it's unusual for "comment" to be used as a verb, but "comment on the code" implies e.g. using GitHub rather than inline code comments